net-misc/ubridge/ubridge-0.9.14.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit fcaps user
+
+DESCRIPTION="Bridge for UDP tunnels, Ethernet, TAP and VMnet interfaces."
+HOMEPAGE="https://github.com/GNS3/ubridge"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+system-iniparser"
+
+RDEPEND="
+	net-libs/libpcap
+	system-iniparser? ( dev-libs/iniparser:* )
```

Comment:
This is a C library, so I seriously doubt `:*` is ever valid here. Have you tested it against slot `4`? If both slots work, it should be `:=` to ensure that it is rebuilt when iniparser is upgraded. Otherwise, it should probably be `:0=` to force slot 0 and rebuild on upgrades within the slot with ABI changes (if those ever happen).